{"uuid":"632aba8b-1ba5-4d58-bd36-0472503f543f","historyId":"9eb35c3546a6609902436c155b7f5527","fullName":"io.cucumber.testng.AbstractTestNGCucumberTests.runScenario","labels":[{"name":"package","value":"com.ual.runner.UcrewRunner"},{"name":"testClass","value":"com.ual.runner.UcrewRunner"},{"name":"testMethod","value":"runScenario"},{"name":"parentSuite","value":"Suite"},{"name":"suite","value":"Test"},{"name":"subSuite","value":"com.ual.runner.UcrewRunner"},{"name":"host","value":"VNDCCORPHP6157"},{"name":"thread","value":"5652@VNDCCORPHP6157.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"}],"links":[],"name":"Runs Cucumber Scenarios","status":"broken","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"java.util.concurrent.ExecutionException: java.lang.NullPointerException: Cannot invoke \"java.lang.Throwable.toString()\" because the return value of \"java.lang.Throwable.getCause()\" is null","trace":"org.testng.TestNGException: java.util.concurrent.ExecutionException: java.lang.NullPointerException: Cannot invoke \"java.lang.Throwable.toString()\" because the return value of \"java.lang.Throwable.getCause()\" is null\r\n\tat org.testng.internal.PoolService.submitTasksAndWait(PoolService.java:54)\r\n\tat org.testng.internal.MethodRunner.runInParallel(MethodRunner.java:127)\r\n\tat org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:819)\r\n\tat org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)\r\n\tat org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)\r\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1511)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:794)\r\n\tat org.testng.TestRunner.run(TestRunner.java:596)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:377)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:276)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1134)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1063)\r\n\tat org.testng.TestNG.run(TestNG.java:1031)\r\n\tat org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)\r\n\tat org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)\r\n\tat org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)\r\nCaused by: java.util.concurrent.ExecutionException: java.lang.NullPointerException: Cannot invoke \"java.lang.Throwable.toString()\" because the return value of \"java.lang.Throwable.getCause()\" is null\r\n\tat java.base/java.util.concurrent.FutureTask.report(FutureTask.java:122)\r\n\tat java.base/java.util.concurrent.FutureTask.get(FutureTask.java:191)\r\n\tat org.testng.internal.PoolService.submitTasksAndWait(PoolService.java:52)\r\n\t... 21 more\r\nCaused by: java.lang.NullPointerException: Cannot invoke \"java.lang.Throwable.toString()\" because the return value of \"java.lang.Throwable.getCause()\" is null\r\n\tat com.ual.hooks.Listener.onTestFailure(Unknown Source)\r\n\tat org.testng.internal.TestListenerHelper.runTestListeners(TestListenerHelper.java:96)\r\n\tat org.testng.internal.TestInvoker.runTestResultListener(TestInvoker.java:219)\r\n\tat org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:656)\r\n\tat org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)\r\n\tat org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:77)\r\n\tat org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:15)\r\n\tat java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)\r\n\tat java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)\r\n\tat java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)\r\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)\r\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)\r\n\tat java.base/java.lang.Thread.run(Thread.java:833)\r\n"},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1697470861484,"stop":1697470861486}